lib/bolt/outputter/human.rb

```diff
@@ -8,6 +8,15 @@ class Human < Bolt::Outputter
                  green: "32",
                  yellow: "33" }.freeze
 
+      BUILTIN_MODULES = %w[
+        puppet_conf
+        package
+        service
+        puppet_agent
+        facts
+      ].freeze
+      BUILTIN_REGEX = %r{\/bolt(-\d+\.\d+\.\d+)?\/modules\/(?:#{Regexp.union(BUILTIN_MODULES).source})}
```

Comment:
I think we should just find the whole path to this directory and use that rather than this regex. We can also skip looking for specific modules and just consider anything in the directory builtin.

Reply:
Is that directory always the same though, no matter what platform you install on or whether you use `gem install` or package? I had modules in both `~/.puppetlabs/bolt/modules` and then when I installed from a package on a new system the modules were at "/opt/puppetlabs/bolt/lib/ruby/gems/2.4.0/gems/bolt-0.23.0/modules/facts"

Reply:
The partial path is probably pretty good: is it in `gems/bolt-<version>/modules`. The rest can depend on platform and whether it's package or gem installed.

Reply:
It will always be the same relative to the rest of bolt right so we should be able to find it absolutely. We rely on that to load it.